src/components/TopNavigation.vue

```diff
@@ -6,6 +6,12 @@
       <a href="/home">Home</a>
     </button>
 
+    <button
+      class="bg-transparent hover:bg-white text-white hover:text-black hover:border-transparent rounded p-4 m-4"
+    >
+      <a href="/story-telling">Story Telling</a>
+    </button>
+
```

Comment:
dont need another navigation option. Just use the existing Home page (`/home` path)